test/functional/page_objects/visualize_page.js

```diff
@@ -364,7 +364,10 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
     async setInspectorTablePageSize(size) {
       const panel = await testSubjects.find('inspectorPanel');
       await find.clickByButtonText('Rows per page: 20', panel);
-      await find.clickByButtonText(`${size} rows`, panel);
+      // Table size buttons are in popover element that is not under the inspectorPanel
+      // but are in an it's own portal attached directly to the body
+      const tableSizesPopover = await find.byCssSelector('.euiPanel');
```

Comment:
Any chance we can use a `data-test-subj` selector here?

Reply:
The buttons are created with EuiBasicTable and I do not think there is any way to attach data-test-subjs to the buttons